content/en/docs/reference/_index.md

```diff
@@ -62,6 +60,9 @@ client libraries:
 * [Scheduler Policies](/docs/reference/scheduling/policies)
 * [Scheduler Profiles](/docs/reference/scheduling/config#profiles)
 
+## Config APIs
```

Comment:
nit:
is this an API or a format? You can't invoke it.

Reply:
This and all other configuration type of definitions are called APIs, although they are not "published" through the API server.

Reply:
I guess it's not hard to change the section title later if we want to.